### PR TITLE
sent messages uses Entities on map View model

### PR DESCRIPTION
**Pull request by teamagam1**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/82" target="_blank">ReviewNinja

File: app/src/test/java/com/teamagam/gimelgimel/app/model/ViewsModels/MessageMapEntitiesViewModelTest.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public void addMessageLatLong_shouldAddMessageAndReturnEntity() throws Exception
         when(geoContent.getPointGeometry()).thenReturn(pg);
```
Comment (BarAgam) [inline app/src/test/java/com/teamagam/gimelgimel/app/model/ViewsModels/MessageMapEntitiesViewModelTest.java:60]: Are there no new tests for the new public method?

File: app/src/main/java/com/teamagam/gimelgimel/app/network/services/GGMessageSender.java [hunk 2]
```diff
@@ -64,10 +62,11 @@ public void sendTextMessageAsync(String message) {
      *
      * @param pointGeometry the message's content location
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/network/services/GGMessageSender.java:63]: adjust method's JavaDoc documentation !fix

File: app/src/test/java/com/teamagam/gimelgimel/app/model/ViewsModels/MessageMapEntitiesViewModelTest.java (hunk 1 above)
Comment (teamagam1) [inline app/src/test/java/com/teamagam/gimelgimel/app/model/ViewsModels/MessageMapEntitiesViewModelTest.java:60]: !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/network/services/GGMessageSender.java (hunk 2 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/network/services/GGMessageSender.java:63]: !fixed

**Pull request merged**